examples/units_and_coordinates/ParkerSolarProbe_trajectory.py

```diff
@@ -0,0 +1,50 @@
+"""
+============================================
+Getting the trajectory of Parker Solar Probe
+============================================
+
+This example shows how to plot the trajectory of Parker Solar Probe (PSP) within the solar system.
+"""
+
+import matplotlib.pyplot as plt
+
+from sunpy.time import parse_time
+import datetime
+
+from sunpy.coordinates import get_body_heliographic_stonyhurst
+from sunpy.coordinates.ephemeris import get_horizons_coord
+
+##############################################################################
+# By using `~sunpy.coordinates.ephemeris.get_horizons_coord`, we can query the 
+# Jet Propulsion Laboratory `Horizons System <https://ssd.jpl.nasa.gov/horizons/>`__ 
+# to get the location of any body they support, but here we will use it to get
+# the coordinates of Parker Solar Probe.
+
+now = parse_time("now")
```

Comment:
```suggestion
# We use :func:`~sunpy.coordinates.get_horizons_coord` to query JPL Horizons
# for the trajectory of Parker Solar Probe (PSP).  Let's request 50 days on
# either side of PSP's 14th closest approach to the Sun.

perihelion_14 = parse_time('2022-12-11 13:16')
```
I don't like the idea of querying with "now" and having the example continuously change with whenever the docs were built.  Let's pin it to the recent perihelion.